module/data/constants.py

```diff
@@ -68,3 +68,10 @@
 
 • RESIDENZA SAN MARZANO
 📞 095330868"""
+
+kb1 = 'Menù mensa 🍽' 
+kb2 = 'Contatti ERSU 📚'
+kb3 = 'Segnalazioni Rappresentanti 📬'
+kb4 = 'help ❔'
+
+keyboard = [[kb1, kb2],[kb3 , kb4]]
```

Comment:
this can be moved into start.py

Reply:
I putted those there so that it can be accessed from main.py and start.py, wouldn't it be better to leave those there?

Reply:
you use only kb2, kb3 and kb4
the `keyboard` variable is used only on `start.py` so it is better to put it there.